### PR TITLE
CI: Add comment with App URL

### PR DESCRIPTION
**Pull request by anxolin**
Adds comment for PRaul: The links still won't work, cause they depend on the PRaul setup and S3 deployment

See the generated comment.

### CODE REVIEW

File: travis/deploy_pull_request.sh [hunk 1]
```diff
@@ -39,9 +39,9 @@ function publish_pull_request_urls_in_github {
   else
     REVIEW_FEATURE_MESSAGE="\
 {\"body\": \"Travis automatic deployment:\\n \
-  * **🔭 [Explorer Dapp]($REVIEW_FEATURE_URL)**: Explorer test app\\n \
-  * **📈 [Trade Dapp]($REVIEW_FEATURE_URL/trade.html)**: Trade test app\\n \
-  * **📚 [Storybook](${REVIEW_FEATURE_URL}/storybook/)**: Component stories\""
+  * **🔭 [Explorer Dapp](${{ REVIEW_FEATURE_URL }})**: Explorer test app\\n \
+  * **📈 [Trade Dapp](${{ REVIEW_FEATURE_URL }})**: Trade test app\\n \
```
Comment (alfetopito) [inline travis/deploy_pull_request.sh:43]: Intentional that both point to the same URL?

**Review by alfetopito — APPROVED** [review]
Approved if we want explorer and trade point to the same endpoint.

File: travis/deploy_pull_request.sh (hunk 1 above)
Comment (anxolin) [reply travis/deploy_pull_request.sh:43]: not at all... will fix

Comment (anxolin) [reply travis/deploy_pull_request.sh:43]: mm, that's odd, I think u reviewed on old code or sth. it should look like this

image

Comment (alfetopito) [reply travis/deploy_pull_request.sh:43]: o.O

Comment (alfetopito) [reply travis/deploy_pull_request.sh:43]: Nope, that's the diff gh is giving me still
screenshot_2021-01-08_10-51-39

**Pull request merged**